Functions/Public/New-GitHubGist.ps1

```diff
@@ -10,17 +10,36 @@ function New-GitHubGist {
       
     .Link
     https://trevorsullivan.net
+    http://dotps1.github.io
     https://developer.github.com/v3/gists
     #>
     [CmdletBinding()]
-    param (
+    Param (
+        [Parameter(ValueFromPipeline = $true, Mandatory = $true)]
+        [String[]]$Path,
+        [Parameter(Mandatory = $true)]
+        [String]$Description,
```

Comment:
If the description isn't mandatory, then let's mark it as non-mandatory. Make sure to include comment-based help:

- Synopsis
- Description
- Each parameter needs a user-friendly description